sdk/opa.go

```diff
@@ -348,14 +358,17 @@ func (opa *OPA) Partial(ctx context.Context, options PartialOptions) (*PartialRe
 		Timestamp: options.Now,
 		Input:     &options.Input,
 		Query:     options.Query,
+		Metrics:   options.Metrics,
 	}
 
+	var provenance types.ProvenanceV1
+
 	var pq *rego.PartialQueries
 	decision, err := opa.executeTransaction(
 		ctx,
 		&record,
 		func(s state, result *DecisionResult) {
-			pq, record.InputAST, record.Bundles, record.Error = partial(ctx, partialEvalArgs{
+			pq, provenance, record.InputAST, record.Bundles, record.Error = partial(ctx, partialEvalArgs{
```

Comment:
I feel a little unsure about how to approach this list and its ordering. For now, I've gone with this order, but open to other ideas.